src/test/scala/com/ambiata/disorder/ListSSpec.scala

```diff
@@ -0,0 +1,16 @@
+package com.ambiata.disorder
+
+import org.specs2.{ScalaCheck, Specification}
+
+class ListSSpec extends Specification with ScalaCheck { def is = s2"""
+
+  ListS properties are met:
+
+    ${ prop((s: ListS) => s.value.filterNot(_.contains("\\s")).length ==== s.value.length) }
+
+    ${ prop((s: ListS) => s.value.filterNot(_.contains("\\n")).length ==== s.value.length) }
```

Comment:
I'm a little confused - this is the (literal) "\n" string? Is this what you mean to check?